src/models/application.py

```diff
@@ -31,6 +31,10 @@ def get_by_user(cls, user_id):
     def get_by_company(cls, company_id):
         return cls.objects.filter(job__company_id=company_id).filter(job__status__lt=2)
 
+    @classmethod
+    def get_by_category(cls, job_category_id):
+        return cls.objects.filter(job__category_id=job_category_id).filter(job__status__lt=2)
```

Comment:
Why the filter `.filter(job__status__lt=2)`?

Reply:
this is old, it works. It is identical to order by company, which you don't complain about so why is this wrong?